### PR TITLE
copy rules for Skill Mastery (Rogue) into Skill Mastery (Investigator)

### LINKED ISSUE #7288: Skill Mastery (Investigator) present, not implemented
The feat Skill Mastery (Investigator) is [present](https://github.com/foundryvtt/pf2e/blob/master/packs/data/feats.db/skill-mastery-investigator.json), but does not implement any rules to make it function. Skill Mastery (Rogue), however, appears to be [fully implemented](https://github.com/foundryvtt/pf2e/blob/master/packs/data/feats.db/skill-mastery-rogue.json) (it worked the way I expected when I tried it).

Please have a look at implementing the necessary logic for the Investigator variant of this feat.

Thanks for all of your hard work on this!

### PR DESCRIPTION
**Pull request by asuffield**
fixes #7288

### CODE REVIEW

File: packs/data/feats.db/skill-mastery-investigator.json [hunk 1]
```diff
@@ -27,7 +27,277 @@
...
+                "allowedDrops": {
+                    "label": "PF2E.SpecificRule.SkillMastery.AllowedDrops",
+                    "predicate": [
+                        "item:trait:skill"
+                    ]
+                },
+                "choices": {
+                    "query": "{\"system.traits.value\":{\"$elemMatch\":\"skill\"}}"
+                },
+                "flag": "skillMasteryRogue",
```
Comment (simonward) [inline packs/data/feats.db/skill-mastery-investigator.json:292]: Flag should probably be skillMasteryInvestigator

File: packs/data/feats.db/skill-mastery-investigator.json [hunk 2]
```diff
@@ -27,7 +27,277 @@
...
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.SkillFeat"
+            },
+            {
+                "key": "GrantItem",
+                "uuid": "{item|flags.pf2e.rulesSelections.skillMasteryRogue}"
```
Comment (simonward) [inline packs/data/feats.db/skill-mastery-investigator.json:298]: "{item|flags.pf2e.rulesSelections. skillMasteryInvestigator}"

File: packs/data/feats.db/skill-mastery-investigator.json (hunk 1 above)
Comment (asuffield) [reply packs/data/feats.db/skill-mastery-investigator.json]: Good catch, fixed

**Comment by intrand** [comment]
Thanks for bothering to implement this!

**Pull request merged**